hazelcast/proxy/reliable_topic.py

```diff
@@ -240,7 +240,13 @@ def _handle_next_batch(self, future):
                     member = None
                     if message.publisher_address:
                         member = MemberInfo(
-                            message.publisher_address, None, None, False, _UNKNOWN_MEMBER_VERSION
+                            message.publisher_address,
+                            None,
+                            None,
+                            False,
+                            _UNKNOWN_MEMBER_VERSION,
+                            None,
+                            {},
```

Comment:
In java, address map of this memberInfo is set to a map that includes single entry: member endpoint qualifier -> address